src/generated/gost_779_alt.ts

```diff
@@ -33,22 +33,17 @@ export default {
         ъ: "``",
         ы: "y`",
         ь: "`",
-        э: "е`",
+        э: "e`",
```

Comment:
Strange

Reply:
It OK

Reply:
![nice](https://user-images.githubusercontent.com/242577/150655113-4544b69e-3e2a-4129-8e01-dff07a09e12e.png)